src/standalone/Vegetation/stomatalconductance.jl

```diff
@@ -19,23 +19,6 @@ struct MedlynConductanceParameters{FT <: AbstractFloat}
     g1::FT
 end
 
-"""
-    function MedlynConductanceParameters{FT}(;
-        Drel = FT(1.6), # unitless
-        g0 =  FT(1e-4), # mol/m^2/s 
-        g1 = FT(790) # converted from 5 √kPa to units of √Pa
-) where{FT}
-
-A constructor supplying default values for the MedlynConductanceParameters struct.
-"""
-function MedlynConductanceParameters{FT}(;
-    Drel = FT(1.6),
-    g0 = FT(1e-4),
-    g1 = FT(790),
```

Comment:
I can't find a counterpart for `g1` in ClimaParams - has it been added there?

Reply:
Right, there is no default for `g1` in ClimaParams, I think @kmdeck said that we should store the default in ClimaLand.

Reply:
yes! because that one is plant-type specific/location dependent.